src/glossary.rst

```diff
@@ -4713,6 +4722,18 @@ opt-out trait bound
 An :dt:`opt-out trait bound` is a :t:`trait bound` with :s:`Punctuation` ``?``
 that nullifies an implicitly added :t:`trait bound`.
 
+
+.. _fls_LnPDQW3bnNUw:
+
+or-pattern
+^^^^^^^^^^
+
+:dp:`fls_LnPDQW3bnNUw`
+An :dt:`or pattern` is a :t:`pattern` that matches on one of two or more :t:`[pattern-without-alternation]s` and or-s two or more :t:`[subpattern]s` using character 0x7C (vertical line).
```

Comment:
Maybe

```suggestion
An :dt:`or pattern` is a :t:`pattern` that matches on one of two or more :t:`[pattern-without-alternation]s` and or-s them using character 0x7C (vertical line).
```

I realized that `pattern-without-alternation` and `subpattern` are not interchangeable, as the latter implies a pattern nested within another pattern at any depth. In the case of `or-pattern`s, the `pattern-without-alternation`s are at the same top level depth.

I suspect that we have other such misuses in the Patterns chapter 😢